internal/backend/oauth/oauth.go

```diff
@@ -475,6 +475,18 @@ func New(l *zap.Logger, vars sdkservices.Vars) sdkservices.OAuth {
 					// "Teamwork.Migrate.All", // Application-only.
 				},
 			},
+			"salesforce": {
```

Comment:
Do you have a better link?
```suggestion

            // https://help.salesforce.com/s/articleView?id=xcloud.remoteaccess_oauth_flows.htm
			// https://help.salesforce.com/s/articleView?id=xcloud.remoteaccess_oauth_endpoints.htm
			"salesforce": {
```

Reply:
Done